static/app/views/eventsV2/results.tsx

```diff
@@ -118,6 +118,7 @@ class Results extends React.Component<Props, State> {
       this.hasChartParametersChanged(prevState.eventView, eventView)
     ) {
       api.clear();
+      loadOrganizationTags(api, organization.slug, selection);
```

Comment:
Would it make more sense to use a separate API client for `loadOrganizationTags` so it doesn't get canceled? Reloading tags on update will result in the tags being loaded every time the `eventView` changes which is rather expensive since the tags endpoint is one of the slowest API calls on this page.

Reply:
Yep. Kevan suggested this yesterday. I updated to instantiate a dedicated `Client` instance for the org tags endpoint.